### DISCUSSION TITLE
Can't make faces any bigger

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#123703] [Mobile]
Let's say you're reading an interesting post. Let's say you're reading this post. And you see my interesting face. Well would you like to get a close look at what I look like? Okay so we tap that face. And indeed we get to see a bigger version. But that's about it, and it's not that big compared to the one we were looking at either. 

So even though we're now  looking at somebody else's profile, there's just no way to make their face bigger, no matter how much we spread our fingers or whatever.

**Comment by mattruggio** [comment]
Hey @jidanni, thanks for the suggestion! I am pretty sure I have used this exact functionality in other (non-GitHub) mobile apps before. I cannot guarantee this request will get fulfilled but I did forward this internally to the mobile team to take a look at.

**Reply by jidanni** [reply]
Thanks Matt. At about the 1 cm maximum size, I can tell that you got light skin and seem to be a male around 30 years old. But I can't tell your ethnicity etc. it's just too small. But maybe we're not supposed to be thinking about too many personal details.
